index.html

```diff
@@ -6,6 +6,42 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="description" content="PyCon India, the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community and it attracts the best Python programmers from across the country and abroad.">
   <meta name="author" content="">
+  <!-- Structured Data markup starts -->
+  <script type="application/ld+json">{"@context":"http://schema.org","@type":"Organization","name":"PyCon India","url":"https://in.pycon.org/2018/","logo":"https://in.pycon.org/2018/img/banner-logo.png","sameAs":["https://twitter.com/pyconindia","https://www.facebook.com/PyConIndia/","https://www.linkedin.com/company/pycon-india/","https://www.linkedin.com/groups/3203762"],"areaServed":{"@type":"Country","name":"India"},"contactPoint":[{"@type":"ContactPoint","email":"contact@in.pycon.org","contactType":"Customer Support","url":"https://in.pycon.org/2018/"}]}</script>
+  <script type="application/ld+json">{"@context":"http://schema.org","@type":"WebSite","about":"PyCon India is the primary Python conference in India. This year's conference is happening on October 05-09, 2018. Overall Overview - PyCon India is the primary Python conference in India. A purely volunteer effort, it is being hosted once an year in India, and will attract some of the best Python developers in India and abroad.","inLanguage":"English","publisher":"PyCon India","potentialAction":{"@type":"SearchAction","target":"https://in.pycon.org/2018/#s={search_term_string}","query-input":"required name=search_term_string"},"keywords":["python","community","conference","pycon","pyconindia","pyconindia2018","foss","pycon hyderabad","python conference","india","tickets","events","2018","proposals","schedule","github","team","twitter","registration","cfp","discount","coupon","corporate discount","hyderabad","python event hyderabad","pycon india facebook","pycon india twitter","pycon india shcedule","pycon india sponsors","pycon india tickets","pycon india cfp","pycon india cfp 2018","pycon india mailing list","pycon india 2018 tickets","pycon india github","pycon india 2018 hyderabad","python community conference","student conference","foss events india"],"copyrightHolder":[{"@type":"Organization","url":"https://in.pycon.org/2018/","name":"PyCon India 2018"}],"name":"PyCon India 2018 ","alternateName":"PyCon India","url":"https://in.pycon.org/2018/"}</script>
+  <div itemscope itemtype="http://www.schema.org/SiteNavigationElement">
```

Comment:
Url should be https://schema.org/SiteNavigationElement

Reply:
No, it's correct. The current schema page doesn't have https.

http://schema.org/SiteNavigationElement

https://prnt.sc/jtivb5

Reply:
Ah I meant removing the www. The url is wrong now.

Reply:
Fixed.